app/services/question_saver.rb

```diff
@@ -19,6 +19,7 @@ def persist_answers(form)
     persist_room(form)
     persist_sor_code(form)
     persist_last_question(form)
+    persist_problem(form)
```

Comment:
Is it worth removing the `persist_last_question` method, if we're no longer using this within the metadata part of the description..?

Reply:
It's probably worth getting rid of, yes. I'm happy to do that as a follow up.